### PR TITLE
final iteration on readme

### PR DESCRIPTION
**Pull request by echohack**
Signed-off-by: David Echo <echohack@users.noreply.github.com>

This is the final iteration on this readme for now, and now drives users into the examples directory, where they can select from the correct package and workflow that they desire.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,124 +24,24 @@ What can you do with Effortless Infrastructure?
...
 
-1. Effortless uses a strong build process for your cookbooks. The build creates a single, deployable package with your cookbooks, an up-to-date Chef Infra Client, and the latest best practices.
+1. Effortless uses a build process to pull down all your cookbooks or profiles. The build creates a single, deployable package. For Chef Infra, it contains your cookbooks, an up-to-date Chef Infra client, and the latest best practices. For Chef InSpec, it contains your profiles, an up-to-date Chef InSpec client, and the latest best practices.
 
 2. At runtime, Chef Infra works without Chef Infra Server. It uses Chef Solo mode.
 
-3. Chef Habitat manages Chef Infra, and provides a pull-based update strategy for continuous delivery of the Chef Infra Client and your cookbooks.
+3. At runtime, Chef InSpec works without pulling profiles from Chef Automate. All profiles, including those from Chef Automate, are vendored at build time.
+
+3. Chef Habitat manages Chef Infra and Chef InSpec, and provides a pull-based update strategy for continuous delivery.
```
Comment (TheLunaticScripter) [inline README.md]: Need to fix the numbering here.

**Comment by jmassardo** [comment]
Shouldn't the configuration section stay in the main readme since it applies to effortless as a pattern and not specifically to the windows audit example?

**Comment by echohack** [comment]
> Shouldn't the configuration section stay in the main readme since it applies to effortless as a pattern and not specifically to the windows audit example?

Each `example` might encapsulate a different workflow, such as the `infra-linux-policyfile-cookbook` example, which is a local dev example using test kitchen.

You're super right in that it does belong in each example where it's relevant, so I've fixed that.

The README.md is an important file for the repo. It's where we tell the story about the software, and deliver call(s) to action for users. It's a good idea to keep all the configuration knobs out of the README.md.

Ultimately I'd like to see docs like this land in a hugo-style website, generated from the `docs` directory, and working together with Chef Software's docs team to make that official. Having an official API enumerated out will help this too.

File: README.md (hunk 1 above)
Comment (echohack) [reply README.md]: Doh. I fixed it forever using markdown's smarts.

**Pull request merged**